### PR TITLE
deps: bump wasmtime => 6.0.0

### PR DESCRIPTION
**Pull request by RiverPhillips**

### CODE REVIEW

File: bazel/cargo/wasmtime/remote/BUILD.rustix-0.35.13.bazel [hunk 1]
```diff
@@ -163,36 +163,20 @@ rust_library(
...
-            "@wasmtime__linux_raw_sys__0_0_46//:linux_raw_sys",
-        ],
-        "//conditions:default": [],
-    }) + selects.with_or({
-        # cfg(any(rustix_use_libc, miri, not(all(target_os = "linux", any(target_arch = "x86", all(target_arch = "x86_64", target_pointer_width = "64"), all(target_endian = "little", any(target_arch = "arm", all(target_arch = "aarch64", target_pointer_width = "64"), target_arch = "powerpc64", target_arch = "riscv64", target_arch = "mips", target_arch = "mips64")))))))
-        (
-            "@rules_rust//rust/platform:i686-unknown-linux-gnu",
-            "@rules_rust//rust/platform:x86_64-unknown-linux-gnu",
-            "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
-            "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
```
Comment (PiotrSikora) [inline bazel/cargo/wasmtime/remote/BUILD.rustix-0.35.13.bazel:176]: This shouldn't be removed. That's why you're seeing all those errors about missing `linux_raw_sys`.

Always try to verify the diff between the old and new version of the patch. That's the easiest way to spot issues.

**Review by PiotrSikora — COMMENTED** [review]
Wasmtime usually bleeding edge Rust features, so you might need to update `rules_rust` to latest version to pull a more recent version of Rust for Wasmtime v6.0.0 to compile.

**Comment by PiotrSikora** [comment]
@RiverPhillips please look at the diff between existing and new versions of Bazel rules for `errno` and `rustix` crates, you can clearly see the added or removed blocks that are breaking the build.

**Comment by RiverPhillips** [comment]
I'm going to close this an re-open a new PR for 6.0.1

**Pull request closed without merging**